client/src/components/Feed/FiltersList.js

```diff
@@ -32,23 +32,25 @@ const FiltersList = () => {
             : location?.address}
         </ButtonTag>
       )}
-      {Object.keys(selectedOptions).map((filter) =>
-        selectedOptions[filter].map((option, idx) => (
-          <ButtonTag
-            key={idx}
-            onClick={handleOption(filter, option)}
-            className="tag-closable"
-          >
-            {t(
-              getOptionText(
-                filters,
-                filter,
-                option.value ? option.value : option,
-              ),
-            )}
-          </ButtonTag>
-        )),
-      )}
+      {Object.keys(selectedOptions)
+        .filter((key) => key !== "lookingFor")
+        .map((filter) =>
```

Comment:
triple iteration? perhaps we are not using the right data structure for `selectedOptions`